model/service_mgmt/v1/root_resource.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Root of the tree of resources of the Management Services.
+resource Root {
+	// Reference to the resource that manages the collection of Managed Services.
+	locator Service {
```

Comment:
This is a collection, should be `services`. Are you sure you want URLs to be `/api/service_mmt/v1/service/123...`?